Automattic/wp-calypso
# Jetpack Cloud: Create Fix all threats - add credentials modal

#### Changes proposed in this Pull Request

- The goal of this is to create the Threat Card used on the main screen when threats are found.

#### Important note

We are not checking if the provided credentials are valid. We move the user to next step in the progress regardless of the result. This is for testing purposes only.

#### Testing instructions

- Navigate to http://jetpack.cloud.localhost:3000/scan/<yourSiteSlug>?scan-state=threats
- Click the "Open Fix All Threats Dialog" button

See 1151678672052943-as-1165787355206381.

#### Expected result
##### We need credentials

<img width="832" alt="image" src="https://user-images.githubusercontent.com/3418513/76857090-81e34f00-6833-11ea-8a61-a643d71f5e27.png">

##### If credentials were already supplied

<img width="827" alt="image" src="https://user-images.githubusercontent.com/3418513/76857102-87409980-6833-11ea-9d92-04c20662f0af.png">

##### Fixing all threats (notice that the buttons are disabled)
![Kapture 2020-03-25 at 10 38 51](https://user-images.githubusercontent.com/3418513/77542454-f5fda280-6e84-11ea-9b84-abb77dc4fb93.gif)
